drivers/clk/sunxi/clk_ncatv2.h

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (C) 2021 Moteen Shah <moteenshah.02@gmail.com>
```

Comment:
```suggestion
 * Copyright (C) 2022 Moteen Shah <moteenshah.02@gmail.com>
```

Also, did you author this file entirely, or is it based on Samuel Holland's implementation?

Reply:
Samuel added the clock header file for sun50i_h6 because D1 utilises the same macros; however, if the additions are to be made generic, the macros must be defined in a distinct header file unique to D1; therefore, should I put Samuel or the original creator of the sun50i_h6 header in the copyright?

Reply:
> Samuel added the clock header file for sun50i_h6 because D1 utilises the same macros; however, if the additions are to be made generic, the macros must be defined in a distinct header file unique to D1; therefore, should I put Samuel or the original creator of the sun50i_h6 header in the copyright?

I think you need to include the copyright of everyone involved (as far as you understand it).